packages/hardhat-core/test/internal/hardhat-network/jsonrpc/client.ts

```diff
@@ -74,6 +74,7 @@ describe("JsonRpcClient", () => {
           fakeProvider as any,
           1,
           123,
+          1,
```

Comment:
I don't understand why this chainId couldn't be 'undefined', but the test returned a slightly incorrect storage value when the chainId wasn't `1`

Reply:
nevermind, I was being a clown :D